_data/contributors.yml

```diff
@@ -233,3 +233,23 @@
   packages-submitted: ["pyrolite"]
   packages-reviewed: [""]
   packages-editor: [""]
+- name: Philip Meier
+  bio: 'PhD student working on Neural Style Transfer'
+  organization: "inIT - Institute Industrial IT, Technische Hochschule Ostwestfalen-Lippe (TH OWL)"
+  github_username: pmeier
+  github_image_id: 6849766
+  contributor_type:
+    - package-maintainer
+  packages-submitted: ["pystiche"]
+  packages-reviewed: [""]
+  packages-editor: [""]
+- name: Edgar Arriba
```

Comment:
```suggestion
- name: Edgar Riba
```

Reply:
Thank you for catching this! Sorry @edgarriba 😬